index.html

```diff
@@ -160,6 +147,26 @@ <h4>Configuration</h4>
 
         </section>
 
+        <section class="row">
+            <h2>Lazy Loading</h2>
+            <p class="col-12">By default, the request for each URI includes documents that annotate it using the Web Annotation pattern. This adds a minimal lag to the query into RERUM, but 
+                may be longer for objects with a very long history or wide network of annotations. In these cases, the <code>deer-lazy</code> attribute may be added. In this case, the 
+                object at the URI will be rendered immediately and then updated with further annotations as they are loaded.
+            </p>
+            <div class="card col-6">
+                <header>Lazy Loading</header>
+                <deer-view deer-id="http://devstore.rerum.io/v1/id/5c9d155fe4b0a44e13e61706"></deer-view>
```

Comment:
I changed this id to http://devstore.rerum.io/v1/id/62e8246b20274674d431540c.  It did not work, the Annotations were not found.  Here is one of those annotations, so you know it exists http://devstore.rerum.io/v1/id/62e8246b20274674d4315418

Reply:
This is confirmed. There are a bunch loaded, but it doesn't come through. I'll check it out specifically.